runtime/doc/usr_41.txt

```diff
@@ -1036,6 +1036,7 @@ Mappings and Menus:			    *mapping-functions*
 	mapset()		restore a mapping
 	menu_info()		get information about a menu item
 	wildmenumode()		check if the wildmode is active
+	wildtrigger()		start wildcard expansion
```

Comment:
Why isn't it named `wildexpand()` or `expandwild()` ? That would be consistent with `expand()` and `expandcmd()`. @chrisbra

Reply:
I think if it's named with `expand`, one may think that it returns a list of matches instead of triggering completion, but that what `getcompletion()` does.

Reply:
it should start with `wild` and I thought about the naming for a bit but couldn't find a better name (or more consistent one). Not sure if `wildexpand()` does this, as we are not expanding anything